### PR TITLE
add sticky anchor links on the assets page

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

- Added the sticky anchor links on the assets page
- It appears only Shibuya for now

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**

Screenshot 2024-01-22 at 6 38 20 PM

### CODE REVIEW

**Comment by Kahonnohak** [comment]
This is all great, lets not spend more time on this but just two points
1) Please change the position just below the account.
2) Background: Gray1
3) Please add the dark theme with the background colour: Navy 3

**Comment by ayumitk** [comment]
@Kahonnohak Thank you for your review! I've updated all of them

**Review by Kahonnohak — APPROVED** [review]
Thank you! Great work - we will update this tab and add notification next time.

File: src/components/assets/Account.vue [hunk 1]
```diff
@@ -290,6 +276,8 @@ export default defineComponent({
 
     const currentNetworkName = ref<string>(providerEndpoints[currentNetworkIdx.value].displayName);
 
+    const { isDappStakingV3 } = useDappStaking();
```
Comment (impelcrypto) [inline src/components/assets/Account.vue:279]: Do you need this line in this file? It seems `isDappStakingV3` hasn't been used.

**Review by impelcrypto — CHANGES_REQUESTED** [review]

File: src/components/assets/Account.vue (hunk 1 above)
Comment (ayumitk) [reply src/components/assets/Account.vue]: @impelcrypto thank you! I forgot to remove it. Fixed in [677420c](https://github.com/AstarNetwork/astar-apps/pull/1149/commits/677420c5de7c1916855cbd9f2ed1819cd2bd9fc9)

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**